sdk/core/azure-core-amqp/README.md

```diff
@@ -11,12 +11,12 @@ own AMQP client library that abstracts from the underlying transport library's i
 
 ### Include the package
 
-[//]: # ({x-version-update-start;com.azure:azure-core-amqp;current})
+[//]: # ({x-version-update-start;com.azure:azure-core-amqp;dependency})
```

Comment:
@alzimmermsft Could you confirm if core packages keep this as current version?

Reply:
You're right. Changed back to current.